cmd/src/campaign_progress_printer_test.go

```diff
@@ -0,0 +1,249 @@
+package main
+
+import (
+	"runtime"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/src-cli/internal/campaigns"
+	"github.com/sourcegraph/src-cli/internal/output"
+)
+
+const progressPrinterDiff = `diff --git README.md README.md
+new file mode 100644
+index 0000000..3363c39
+--- /dev/null
++++ README.md
+@@ -0,0 +1,3 @@
++# README
++
++This is the readme
+diff --git a/b/c/c.txt a/b/c/c.txt
+deleted file mode 100644
+index 5da75cf..0000000
+--- a/b/c/c.txt
++++ /dev/null
+@@ -1 +0,0 @@
+-this is c
+diff --git x/x.txt x/x.txt
+index 627c2ae..88f1836 100644
+--- x/x.txt
++++ x/x.txt
+@@ -1 +1 @@
+-this is x
++this is x (or is it?)
+`
+
+func TestCampaignProgressPrinterIntegration(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Something emits different escape codes on windows.")
```

Comment:
Can you create a follow up issue for us to figure this out at some point, please? I don't really want to leave this skipped forever, and there _should_ be a way to use the same approach on Windows.

Reply:
Done! Yeah, I also think so. https://github.com/sourcegraph/sourcegraph/issues/16353